scripts/patterns-reorg/update_qiskit_bot.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3.11
```

Comment:
This script, together with update_internal_links, is meant to be used only one time. After the migration, they will be removed